src/main/kotlin/no/nav/syfo/behandler/BehandlerDialogmeldingService.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.syfo.behandler
+
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.behandler.database.*
+import no.nav.syfo.behandler.database.domain.toBehandlerDialogmeldingBestilling
+import no.nav.syfo.behandler.domain.BehandlerDialogmeldingBestilling
+import java.util.UUID
+
+class BehandlerDialogmeldingService(
+    private val database: DatabaseInterface,
+) {
+    fun getDialogmeldingBestillingListe(): List<BehandlerDialogmeldingBestilling> {
+        return database.getBehandlerDialogmeldingBestillingNotSendt()
+            .map { pBehandlerDialogMeldingBestilling ->
+                pBehandlerDialogMeldingBestilling.toBehandlerDialogmeldingBestilling(
+                    database.getBehandlerDialogmeldingForId(pBehandlerDialogMeldingBestilling.behandlerId)!!.behandlerRef
+                )
+            }
+    }
+
```

Comment:
Ser vi gjør litt forskjellig rundt omkring, men siden man uansett commiter i metodene under så kunne man kanskje gjort de til extension-metoder på DatabaseInterface i stedet for Connection i BehandlerDialogmeldingBestillingQuery

Reply:
Ja, enig i at det er bedre i disse tilfellene. Fikser.